Nare_Mkrtchyan/python/25_05/fibonachi.py

```diff
@@ -0,0 +1,34 @@
+def check(num):
+"""
+    Check function checks if user input is number or not. If it is number returns true, otherwise returns false. 
+ """
+    try:
+        int(num)
+        return True
+    except ValueError:
+        return False
+
+def fibonachi(n):
+    """
+    fibonachi function returns n'th fibonachi number.
+    """
+    if(n==1):
+        return 1
+    if(n==2):
+        return 1
+    if(n>2):
+        return fibonachi(n-1)+fibonachi(n-2)
+
+def start():
+    """
+    start function calls check and fibonachi functions
+    """
+    num=input()
+    if check(num):
+        num=int(num)
+        print(num,"'th fibonachi item is: ", fibonachi(num))
+    else:
+        print(num,"is not number. Enter number.")
+        start()
+
+start()
```

Comment:
1. ![image](https://user-images.githubusercontent.com/80816144/119564585-ff9cd600-bdb9-11eb-86e8-00a6fa8e4592.png)

2. F(0) -> 0
F(1) -> 1 etc
https://en.wikipedia.org/wiki/Fibonacci_number


![image](https://user-images.githubusercontent.com/80816144/119564811-3d016380-bdba-11eb-8eca-508a2769cdbf.png)

3. 
![image](https://user-images.githubusercontent.com/80816144/119564953-65895d80-bdba-11eb-9f38-841695f31d48.png)

Reply:
Tried to do marked changes.